.gitignore

```diff
@@ -27,7 +27,6 @@ xcuserdata
 /platform/ios/benchmark/assets/glyphs/DIN*
 /platform/ios/benchmark/assets/tiles/mapbox.mapbox-terrain-v2,mapbox.mapbox-streets-v6
 /platform/darwin/developer.xcconfig
-**/token
```

Comment:
Could we add this back? I think it was added for security reasons, so that we don't commit tokens accidentally.

Reply:
I see, I will revert this one. Thanks!